src/responder/common/responder_common.c

```diff
@@ -1194,6 +1194,17 @@ int sss_process_init(TALLOC_CTX *mem_ctx,
         }
     }
 
+    ret = confdb_get_string_as_list(rctx->cdb, rctx,
+                                    CONFDB_MONITOR_CONF_ENTRY,
+                                    CONFDB_MONITOR_LOOKUP_ORDER,
+                                    &rctx->lookup_order_list);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_MINOR_FAILURE,
```

Comment:
I think you should special case ENOENT or use a higher debug level because a missing lookup_order is not an error.